akka-stream/src/main/scala/akka/stream/ActorFlowMaterializer.scala

```diff
@@ -163,6 +165,13 @@ abstract class ActorFlowMaterializer extends FlowMaterializer {
  */
 class MaterializationException(msg: String, cause: Throwable = null) extends RuntimeException(msg, cause)
 
+/**
+ * This exception signals that an actor implementing a Reactive Streams Subscriber, Publisher or Processor
+ * has been terminated without being notified by an onError, onComplete or cancel signal. This usually happens
+ * when an ActorSystem is shut down while stream processing actors are still running.
+ */
+object AbruptTerminationException extends Exception("Processor actor terminated abruptly") with NoStackTrace
```

Comment:
true. I think it should also be a class instead of object and contain the path of the actor that throws it